packages/web-app-admin-settings/src/composables/actions/groups/useGroupActionsDelete.ts

```diff
@@ -0,0 +1,100 @@
+import { computed } from 'vue'
+import { Store } from 'vuex'
+import { eventBus } from 'web-pkg'
+import { useClientService, useStore } from 'web-pkg/src/composables'
+import { GroupAction, GroupActionOptions } from 'web-pkg/src/composables/actions'
+import { useGettext } from 'vue3-gettext'
+
+export const useGroupActionsDelete = ({ store }: { store?: Store<any> }) => {
+  store = store || useStore()
+  const { $gettext, $ngettext } = useGettext()
+  const clientService = useClientService()
+
+  const deleteGroups = async (groups) => {
+    const graphClient = clientService.graphAuthenticated
+    const promises = groups.map((group) => graphClient.groups.deleteGroup(group.id))
+
+    try {
+      await Promise.all(promises)
```

Comment:
maybe needs a followup, correct me if i'm wrong but given the case that the first two requests succeed and the third fails, it will only alert the user that one group has failed but it will not show that two have been deleted.

Reply:
also the `eventBus.publish('app...load')` will not have been called